tincr/io/design/rapidsmith.tcl

```diff
@@ -0,0 +1,238 @@
+package provide tincr.io.design 0.0
+package require Tcl 8.5
+package require tincr.cad.design 0.0
+package require tincr.cad.device 0.0
+package require tincr.cad.util 0.0
+
+namespace eval ::tincr:: {
+    namespace export \
+        run_rapidsmith_command
```

Comment:
Yes. The other methods are only intended to be used by the unit tests in RapidSmith.